webapp/src/Service/ScoreboardService.php

```diff
@@ -259,6 +260,8 @@ public function calculateScoreRow(
 
         $contestStartTime = $contest->getStarttime();
 
+        $pointsJury      = "0";
```

Comment:
Why not `$scoreJury`? We have points already for multipoint problems, so this might be confusing?